### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by Ablu**
I created this pullrequest for him since he is apparently struggling with the git tools :)

### CODE REVIEW

**Comment by Ablu** [comment]
Will review it later.

**Comment by bjorn** [comment]
Awesome, thanks guys! I'll wait till you guys got it reviewed. I think I forgot to mention in my announcement that I was planning to release in two weeks, so you got some time.

**Comment by wayfu** [comment]
I have definitely stopped struggling and given up, to preserve my sanity. :)

File: translations/tiled_de.ts [hunk 1]
```diff
@@ -593,17 +593,17 @@
...
         <location line="+8"/>
         <source>&amp;Never</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nie</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>For &amp;Selected Objects</source>
-        <translation type="unfinished"></translation>
+        <translation>Für ausge&amp;wählte Objekte</translation>
```
Comment (Ablu) [inline translations/tiled_de.ts]: Hm. This changes the accessor key... @bjorn @wayfu wouldn't it make sense to put the `&` before the `s` in the German part too?

File: translations/tiled_de.ts [hunk 2]
```diff
@@ -1348,22 +1348,22 @@ Zeile %1, Spalte %2</translation>
...
         <location line="+12"/>
         <source>Uncaught exception in script. Please check console.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nicht abgefangener Ausnahmefehler im Skript. Bitte Konsole überprüfen.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Script returned false. Please check console.</source>
-        <translation type="unfinished"></translation>
+        <translation>Skript liefert "falsch" zurück. Bitte Konsole überprüfen.</translation>
```
Comment (Ablu) [inline translations/tiled_de.ts]: @wayfu *`lieferte` würde ich sagen...

**Comment by Ablu** [comment]
Looks fine otherwise

File: translations/tiled_de.ts (hunk 1 above)
Comment (bjorn) [inline translations/tiled_de.ts]: Yes, as long as that doesn't conflict with another menu entry.

Comment (wayfu) [inline translations/tiled_de.ts]: I was under the impression that the accessor key was generated from the ampersand in the text and not hardcoded? @bjorn how do you handle it when the original key is not present in the translation? is there a real value in having the same accessor key in all languages, and does it trump the value of having memorizable/sensible accessors?

File: translations/tiled_de.ts (hunk 2 above)
Comment (wayfu) [inline translations/tiled_de.ts]: jo, geht auch

File: translations/tiled_de.ts (hunk 1 above)
Comment (bjorn) [inline translations/tiled_de.ts]: Right, for these accelerators, you should adjust them to the language and just avoid conflicts within the same menu. You guys decide whether **s** or **w** is best here. :-)

Comment (Ablu) [inline translations/tiled_de.ts]: I think sticking to the &S makes it easier for the people who jump 
between the english and german version. At least so far I tried to stick 
to the english accessor keys.

**Comment by Ablu** [comment]
@wayfu are you planning to adapt some of the changes I suggested?

**Comment by wayfu** [comment]
Oh, I thought you were going to do that. Misunderstanding :) Should I send you the file again with the two changes above?

**Comment by Ablu** [comment]
Well I do not mind :) Best would be a git commit, but you can mail me 
the file too, then I can do the commit for you.

**Comment by Ablu** [comment]
Added @wayfu's fixes. Looks good to me now.

**Comment by bjorn** [comment]
Great, thanks!

**Pull request merged**